devSetup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cp .recipes/.tmplr.yml .
+export SCOPE="@patternfly"
+export DESCRIPTION="This library provides patternfly extensions"
+npx tmplr
```

Comment:
Do you think we should keep things like this file and the tmplr file in templated repos? I feel like we probably shouldn't, but I very well might be overlooking something.

Reply:
It should be deleted.